doc/whats_new.rst

```diff
@@ -73,6 +73,8 @@ API
 
 - Add average_dipoles function to `hnn_core.dipole`, by `Blake Caldwell`_ in `#156 <https://github.com/jonescompneurolab/hnn-core/pull/156>`_
 
+- Create a new API for defining external drives and biases to network, by `Christopher Bailey`_ in `#221 <https://github.com/jonescompneurolab/hnn-core/pull/221>`_
```

Comment:
explicitly name the methods you added? And you can link to them in the API documentation for users to look at using :func:Network.add_poisson_drive etc